mon-pix/app/services/session.js

```diff
@@ -122,35 +116,9 @@ export default class CurrentSessionService extends SessionService {
     userIdForLearnerAssociationStorage.remove();
   }
 
-  async _loadCurrentUserAndSetLocale(locale = null) {
+  async _loadCurrentUserAndLocale(language = null) {
```

Comment:
Quel est l’intérêt d’avoir un default parameter `null` dans la déclaration de cette fonction ?

```suggestion
  async _loadCurrentUserAndLocale(language) {
```